ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -339,21 +380,44 @@ open class BadgeView: UIView {
     open override func layoutSubviews() {
         super.layoutSubviews()
         backgroundView.frame = bounds
-        label.frame = bounds.insetBy(dx: -size.horizontalPadding, dy: -size.verticalPadding)
+
+        if let customViewSize = customViewSize(for: frame.size), customViewSize != .zero {
+            customView?.frame = CGRect(origin: CGPoint(x: customViewPadding.left, y: (frame.height - customViewSize.height) / 2), size: customViewSize)
+            label.frame = CGRect(origin: CGPoint(x: customViewPadding.left + customViewPadding.right + customViewSize.width, y: (frame.height - labelSize.height) / 2), size: labelSize)
+        } else {
+            label.frame = bounds.insetBy(dx: -size.horizontalPadding, dy: -size.verticalPadding)
+        }
+
+        flipSubviewsForRTL()
     }
 
     open func reload() {
         label.text = dataSource?.text
         style = dataSource?.style ?? .default
         size = dataSource?.size ?? .medium
+
+        customView?.removeFromSuperview()
+        customView = dataSource?.customView
+        if let customView = customView {
```

Comment:
You can just use the dataSource call directly:

```
        if let customView = dataSource?.customView {
            addSubview(customView)
        }
```

Reply:
Fixed